definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js

```diff
@@ -398,8 +398,8 @@ declare module "antd" {
     okText?: string,
     okType?: 'primary' | 'dashed' | 'ghost' | 'danger' | 'default',
     title?: 'string' | React$Node,
-    onCancel?: (event?: SyntheticEvent<HTMLElement>) => void,
-    onConfirm?: (event?: SyntheticEvent<HTMLElement>) => void,
+    onCancel?: (event: SyntheticEvent<HTMLElement>) => void,
```

Comment:
If we can't know the element type in advance, I would rather use the `SyntheticEvent<>` form.

Reply:
@pascalduez Thanks, changed.